tools/sdk-cli/src/command_handlers.rs

```diff
@@ -99,13 +96,20 @@ pub(crate) async fn handle_command(
             .map(|res| serde_json::to_string_pretty(&res))?
             .map_err(|e| e.into()),
         Commands::ReceivePayment {
-            amount,
+            amount: amount_sats,
             description,
         } => {
-            let invoice = sdk()?.receive_payment(amount, description).await?;
+            let invoice = sdk()?
+                .receive_payment(ReceivePaymentRequest {
+                    amount_sats,
+                    description,
+                    preimage: None,
+                    opening_fee_params: None,
+                })
+                .await?;
             let mut result = serde_json::to_string(&invoice)?;
```

Comment:
Perhaps we also want to print the opening channel absolute fees in case they are exists?

Reply:
We already print them as part of
```rust
            let invoice = sdk()?
                .receive_payment(ReceivePaymentRequest {
                    amount_sats,
                    description,
                    preimage: None,
                    opening_fee_params: None,
                })
                .await?;
            let mut result = serde_json::to_string(&invoice)?;
```

because the `invoice` type is

```rust
pub struct ReceivePaymentResponse {
    pub ln_invoice: LNInvoice,
    pub opening_fee_params: Option<OpeningFeeParams>,
    pub opening_fee_msat: Option<u64>,
}
```